extensions/Worker.Extensions.Tables/src/TableInputAttribute.cs

```diff
@@ -60,7 +72,7 @@ public TableInputAttribute(string tableName, string partitionKey, string rowKey)
         /// <summary>
         /// Used with filter. RowKey should be null. 
         /// </summary>
-        public int? Take
+        public int Take
```

Comment:
Why was the null operator remove here? This may be a breaking change

Reply:
I get an error when I try to use Take like this:
![image](https://user-images.githubusercontent.com/37918412/236585571-a5e8bd13-263d-47dd-a924-efd20f3627f1.png)

The error is `Take is not a valid named attribute argument because it is not a valid attribute parameter type`

Reply:
The error goes away when I remove the nullable. Any ideas for why that happens?

Reply:
Added a new constructor